src/server/plugins/engine/pageControllers/PageControllerBase.ts

```diff
@@ -686,13 +690,26 @@ export class PageControllerBase {
     if (feedbackContextInfo) {
       viewModel.name = feedbackContextInfo.formTitle
     }
+
     // setting the feedbackLink to undefined here for feedback forms prevents the feedback link from being shown
-    if (this.def.feedback?.url) {
-      viewModel.feedbackLink = this.feedbackUrlFromRequest(request)
+    let feedbackLink: string | undefined = this.feedbackUrlFromRequest(request)
```

Comment:
Cheers for simplifying this

It can probably figure out the types from the return value if that helps

```suggestion
    let feedbackLink = this.feedbackUrlFromRequest(request)
```